assignment.md

```diff
@@ -12,46 +12,83 @@ If the version control concepts or `git` commands are still confusing to you, co
+                           
         ```
     
     - c. What was the commit message associated with the last modification to the `-Week 2` line of `index.md` file? (Hint: use `git blame` and `git show`, check out their help pages if you are not sure what to do)
+    added ch2 assignment   
 
  5. In the course website repository (or another repository you want to clone from GitHub), modify one of its existing files, do not `git add` or `git commit` yet:
      - a. What happens when you do `git stash`, be specific about what happens to your uncommitted changes after `stash`.?
-       > Answer here
+       > 'Git stash' saves uncommitted changes from the stash log and removes from working directory
      - b. What do you see when you run `git log --all --oneline` after stashing your modification?
-       > Answer here
+       > The stash is shown first before the log.
      - c. Look up what `git stash drop` and `git stash pop` does, either Google or use `--help`. Run `git stash drop` to undo what you did with `stash`, in what scenario might `stash`-ing, `pop`-ing, and `drop`-ing be useful?
-       > Answer here
+       > Git stash drop removes a single stash entry. Stashing poping and droping may be useful for testing different changes and then deciding what to get rid of with 'drop'. 
 
  6. One common mistake when learning `git` is to commit large files that should not be managed by `git` or adding sensitive information like security keypairs for Amazon Cloud Services. Navigate back to your assignment repository and try adding a random text file to the repository (use `touch`), making some modifications and commits to your repository, and deleting that file with `rm`. But deleting with `rm` will not delete that file's recorded git history, therefore we need to do a little more. You may want to look at [this](https://help.github.com/articles/removing-sensitive-data-from-a-repository/))
```

Comment:
It's not clear what exactly you did in commit f09c2c1006e063c8a18e28f59cd0b9fe702068cb. 

I see you indicate that you removed an empty file, but I don't see a commit where you added the file and recorded your modifications. The goal was to use filter-branch and `git rm` to remove some sensitive content that's been committed. 

I expected 
 - A commit where a sensitive file was added, and some text with fake sensitive information is present in the file.
 - A commit where you deleted these files from your repository. But at this point, the sensitive information is still recorded by git.
 - Another commit where you used `filter-branch` and `git rm` to remove the sensitive text from your commit history. 

-10%